backend/api/Controllers/ReportController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Api.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class ReportController : ControllerBase
+{
+    private readonly ILogger<ReportController> _logger;
+    private readonly ReportService _reportService;
+
+    public ReportController(ILogger<ReportController> logger, ReportService reportService)
+    {
+        _logger = logger;
+        _reportService = reportService;
+    }
+
+    /// <summary>
+    /// List all reports on the asset.
+    /// </summary>
+    /// <remarks>
+    /// Responses are paginated.
+    ///
+    /// Example query:
+    ///
+    ///     /reports?
+    ///
+    /// <para> This query gets all reports (paginated) </para>
+    /// </remarks>
+    /// <returns> List of reports </returns>
+    /// <response code="200"> The list of reports was successfully returned </response>
+    /// <response code="400"> The query is invalid </response>
```

Comment:
Are these necessary when it's documented below?

Reply:
Agreed, fixed